totalopenstation/formats/trimble_are.py

```diff
@@ -25,19 +25,16 @@
 
 class FormatParser(Parser):
 
-    def __init__(self, data):
-        Parser.__init__(self, data)
-
     def is_point(self, line):
         is_point = False
         if "5=" and "4=" and "37=" and "38=" and "39=" in line:
             is_point = True
         return is_point
 
-    def get_point(self, chunk):
+    def get_point(self, line):
```

Comment:
If you look at the sample data, e.g. https://github.com/steko/totalopenstation/blob/master/sample_data/trimble/BSG-08-11-19.are there is one "value" per line and that to obtain a point you have to parse several lines. The `line` variable name seems misleading.

Reply:
Ok, I will revert it.